pypeit/core/wavecal/wvutils.py

```diff
@@ -444,9 +491,12 @@ def xcorr_shift(inspec1,inspec2, smooth=1.0, percent_ceil=80.0, use_raw_arc=Fals
         inspec2 : ndarray
             Spectrum for which the shift and stretch are computed such
             that it will match inspec1
-        smooth: float, default=1.0
-            Gaussian smoothing in pixels applied to both spectra for the
-            computations. Default is 5.0
+        sigdetect (float, optional, default=3.0):
```

Comment:
wrong doc style (or at least different from what is used for `inspec2`)

Reply:
Need help with docs.